app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -28,6 +41,16 @@ def create
       def show
       end
 
+      api :PUT, "/config_templates/:config_template_id/template_combinations/:id", N_("Update template combination")
+      api :PUT, "/hostgroups/:hostgroup_id/template_combinations/:id", N_("Update template combination")
+      api :PUT, "/environments/:environment_id/template_combinations/:id", N_("Update template combination")
```

Comment:
you need to define :config_template_id/hostgroup_id etc.  They're already on the actions above, so I'd suggest creating a param_group and using that on all three actions to avoid copy/pasting it a third time.